### PR TITLE
Change color of box next to move in GameLog

### PR DESCRIPTION
**Pull request by scheijan**
I've changed the code to use the `playerID` from `log` to assign a number to each player. If you think that's too much in one line I can split it up and use a regular `if`. Also: I don't know how you feel about "template strings" in this project. Should I change this to `str()` and `+`?

### CODE REVIEW

File: src/client/log/log.js [hunk 1]
```diff
@@ -59,8 +60,11 @@ export class GameLog extends React.Component {
         currentTurn = [];
       } else {
         const args = item.move.args || [];
+        const playerID = item.move.playerID;
+        const playerNumber = playerIDs.has(playerID) ? [...playerIDs].indexOf(playerID) : playerIDs.add(playerID).size - 1;
```
Comment (nicolodavis) [inline src/client/log/log.js]: Change playerIDs to a map from playerID to playerNumber so you don't have to do this transformation here?

You can do something like:
```
if not in map:
  map[playerID] = map.size()
```

**Comment by nicolodavis** [comment]
@scheijan Could you rebase your branch so that it contains just commits from you? Happy to fix it for you as well (let me know).

**Comment by scheijan** [comment]
I'm not quite sure what I did there... I had a merge conflict and I tried to `git pull --rebase` but I ended up with an "empty" commit. If you know an easy way to fix this please do or I can start over with a new fork.

**Comment by nicolodavis** [comment]
I usually do the following:

1. Go to your master branch.

```
$ git checkout master
$ git pull 
```

2. Go to your feature branch:

```
$ git checkout logColor
$ git rebase master
```

3. This may sometimes have some conflicts. You can fix them and then

```
$ git add <fixed file>
$ git rebase --continue
```

If you are afraid of further corrupting `logColor`, you can experiment locally by creating a copy of `logColor` and trying the above.

```
$ git checkout -b experiment logColor
```

Given that you have some stuff in the branch now that you want to "undo", you have a few options:

```
$ git checkout -b experiment logColor
$ git reset --hard 5a098766150ae2d61510c639867758f57b8d20b1   # pushes the branch back to the first commit
$ git cherry-pick fdea709fbded5d85bcc9bc14724a47ebe7be3298    # cherrypick's your second commit
```

Then rebase your `experiment` branch. If all looks good, you can "copy" your `experiment` branch back to `logColor` and push it back to the PR:

```
$ git checkout logColor
$ git reset --hard 5a098766150ae2d61510c639867758f57b8d20b1
$ git merge experiment   # should be a fast-forward merge
$ git push
```

**Comment by nicolodavis** [comment]
If that doesn't work, let me know and I can fix it later!

**Comment by scheijan** [comment]
I don't know whether this made things any better... I can't see the commit mentioned in Googlebot's "bad news"... If this is too much trouble, I can just close this PR and create a new one from a new branch...

**Comment by nicolodavis** [comment]
No worries! Taking a look now.

**Pull request merged**